src/main.ts

```diff
@@ -7,6 +7,7 @@ import Leaderboard from './components/Leaderboard.vue'
 import Leaderboards from './components/Leaderboards.vue'
 import Main from './components/Main.vue'
 import './style.css'
+import Create from './components/leaderboards/Create.vue'
```

Comment:
Give this a better name, otherwise it'll end up clashing with the create category page.

Reply:
Should I call it `LeaderboardsCreate` or `LeaderboardCreate`? This affects the route and file path, as well

Reply:
Either is fine. I guess I'm partial to `leaderboardCreate`.

Reply:
Alright. I'll change to singular. Also; I'll move the View and List pages into the folder too, as I've mentioned above.